dynamic-plugins/wrappers/backstage-community-plugin-catalog-backend-module-pingidentity/dist-dynamic/package.json

```diff
@@ -22,9 +22,7 @@
     "./package.json": "./package.json"
   },
   "scripts": {},
-  "dependencies": {
-    "@janus-idp/cli": "1.13.1"
-  },
+  "dependencies": {},
```

Comment:
title of this issue is "move to devDependency" but the devDependencies is empty.

Reply:
ah, I see it was done in https://github.com/backstage/community-plugins/pull/1210/files